src/app/api/request-schedule/timeslotHandle.ts

```diff
@@ -0,0 +1,95 @@
+import { prisma } from "@/lib/prisma";
+
+// Define allowed time slots
+const timeSlots = [
+  { start: "08:00", end: "11:00" },
+  { start: "11:00", end: "14:00" },
+  { start: "14:00", end: "17:00" },
+];
```

Comment:
FYi the structure for the timeslot ranges is in src/lib/utils/time-slot/time-slot.ts

The idea of this file is to be a single source of truth for the time slot ranges, so if we decide to change the start and end ranges, we do it in one place, instead of having to search the code for different places where we define this.